packages/tc-ui/relationship/browser.jsx

```diff
@@ -0,0 +1,11 @@
+const { render, h } = require('preact');
+const { RelationshipPicker } = require('./lib/relationship-picker.jsx');
+
+module.exports = {
+	attachEditComponent: container =>
+		render(
+			<RelationshipPicker {...JSON.parse(container.dataset.props)} />,
+			document,
+			container,
+		),
```

Comment:
naming: this looks like HoC component, so the name should be `withEditComponent`, what do you think?

Reply:
what does HoC mean?

Reply:
Ah yes - I read about it. I'll change this in the PR I'm working on now (which is based off this branch) in order to avoid having to rebase.